src/utils/inlineImage.ts

```diff
@@ -0,0 +1,30 @@
+import { SupabaseActions } from '@/utils/SupabaseActions'
+import React from 'react'
+import { generateRandomString } from '@/utils/generateRandomString'
+import { ISignedUrlUpload } from '@/types/interfaces'
+import { getSignedUrlUpload } from '@/app/actions'
+import { getSignedUrlFile, postScrapImage } from '@/app/detail/[task_id]/[user_type]/actions'
+import { ScrapImageRequest } from '@/types/common'
+import { getFilePathFromUrl } from '@/utils/signedUrlReplacer'
+
+export const uploadImageHandler = async (file: File, token: string, task_id?: string): Promise<string> => {
```

Comment:
In some places task_id is nullable, somewhere optional. Can we please keep this consistent?